README.md

```diff
@@ -34,29 +34,49 @@ def handler(event:, context:)
     JSON.generate(body)
 end
 
-SignalFx::Lambda::Tracing.register_handler(&method(:handler))
+SignalFx::Lambda.register_handler(metrics: true, tracing: true, &method(:handler))
 ```
 
 `register_handler` will accept any block.
 
+If passing in a block parameter, it must be the last argument.
+
+It also takes these optional arguments:
+- `metrics`: Enable reporting of metrics. Default: `true`
+- `tracing`: Enable tracing. Default: `true`
+
 ### Tracer configuration
 
 The tracer used by the function is configured through environment variables:
 
 ```
 SIGNALFX_ACCESS_TOKEN
-SIGNALFX_INGEST_URL
 SIGNALFX_SERVICE_NAME
+SIGNALFX_TRACING_URL
 ```
 
-In production, `SIGNALFX_INGEST_URL` should be pointing to your [Smart Gateway](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html).
+In production, `SIGNALFX_TRACING_URL` should be pointing to your [Smart Gateway](https://docs.signalfx.com/en/latest/apm/apm-deployment/smart-gateway.html).
```

Comment:
How does this work with Lambda?  Do you deploy Smart Gateway on an EC2 instance and then give the Lambdas access to it?

Reply:
I think so, but I have not tested with the Smart Gateway

Reply:
Also, the gateway can accept both traces and metrics so it is probably simpler to tell people to just point everything to the gateway.

Reply:
I'll add this with the combined `SIGNALFX_ENDPOINT_URL`